pkg/nfd-master/nfd-master.go

```diff
@@ -716,7 +740,7 @@ func getDynamicValue(value string, features *nfdv1alpha1.Features) (string, erro
 	return element, nil
 }
 
-func filterTaints(taints []corev1.Taint) []corev1.Taint {
+func (m *nfdMaster) filterTaints(taints []corev1.Taint) []corev1.Taint {
```

Comment:
nit: no need to touch this now(?)

```suggestion
func filterTaints(taints []corev1.Taint) []corev1.Taint {
```